ui/raidboss/data/03-hw/dungeon/fractal_continuum.txt

```diff
@@ -1,37 +1,32 @@
 hideall "--reset--"
 hideall "--sync--"
-hideall "--start--"
 
-0 "--reset--" sync /:Exhibit level III is no longer sealed/ window 10000 jump 0
-0 "--reset--" sync /:The high-level incubation bay is no longer sealed/ window 10000 jump 0
-0 "--reset--" sync /:The reality augmentation bay is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /is no longer sealed/ window 10000 jump 0
 
 
 ###Phantom Ray
-#-ii F7D
+#-ii F7D -p F7F:100
 
-0 "Start"
-0.0 "--start--" sync /Exhibit level III will be sealed off/ window 0,1
+0.0 "--sync--" sync /Exhibit level III will be sealed off/ window 0,1
 10.3 "Rapid Sever" sync /:Phantom Ray:F7A:/
-16.6 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+16.6 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/ window 5,5
```

Comment:
For what it's worth, the default window is 5 seconds, so `window 5,5` is a noop.

If you're worried about syncs, I'd say maybe just put like a 10-20 seconds window on Rapid Sever?

Reply:
That's very odd, it's definitely not been operating that way. Is that what's defined in `timeline.js` at line 158? Both there and in DnT's guide it looks like the default is 2.5+/-.

Reply:
Sorry sorry, you are 100% correct.